sktime/forecasting/trend.py

```diff
@@ -207,6 +207,9 @@ def _predict(self, fh=None, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
         y_pred_int : pd.DataFrame
             Prediction intervals for the forecast
         """
+        if return_pred_int:
```

Comment:
please revert this - it has nothing to do with the PR, and the `NotImplementedError` is dealt with by the base class

Reply:
I think I can remove this `raise` completely then, rather then reverting?

Reply:
ok I think this is resolved now as I removed it completely, which seems to be the `main` state as well.